car_information/label.py

```diff
@@ -0,0 +1,77 @@
+import os
+import shutil
+from car_information_service import get_data_from_vegvesenet, \
+    identify_license_plate
+import sys
+
+
+def safe_directory(directory):
+    os.makedirs(directory, exist_ok=True)
+
+
+def copy_image(image, input_directory, output_directory):
+    safe_directory(output_directory)
+    shutil.copy(input_directory + image, output_directory + image)
+
+
+def delete_file(filepath):
+    os.unlink(filepath)
+
+
+def init_output_directory(directory):
+    safe_directory(directory)
+    safe_directory(directory + "/color")
+    safe_directory(directory + "/brand")
+    safe_directory(directory + "/model")
+    safe_directory(directory + "/unlabelled")
```

Comment:
Could also use `os.path.join`

Reply:
Thanks! Yeah it feels more natural this way.